tests/baselines/reference/typeAssertions.errors.txt

```diff
@@ -1,12 +1,8 @@
 tests/cases/conformance/expressions/typeAssertions/typeAssertions.ts(5,9): error TS2558: Expected 0 type arguments, but got 1.
-tests/cases/conformance/expressions/typeAssertions/typeAssertions.ts(31,12): error TS2352: Conversion of type 'SomeOther' to type 'SomeBase' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
-  Property 'p' is missing in type 'SomeOther'.
-tests/cases/conformance/expressions/typeAssertions/typeAssertions.ts(35,15): error TS2352: Conversion of type 'SomeOther' to type 'SomeDerived' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
-  Property 'x' is missing in type 'SomeOther'.
-tests/cases/conformance/expressions/typeAssertions/typeAssertions.ts(37,13): error TS2352: Conversion of type 'SomeDerived' to type 'SomeOther' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
-  Property 'q' is missing in type 'SomeDerived'.
-tests/cases/conformance/expressions/typeAssertions/typeAssertions.ts(38,13): error TS2352: Conversion of type 'SomeBase' to type 'SomeOther' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
-  Property 'q' is missing in type 'SomeBase'.
+tests/cases/conformance/expressions/typeAssertions/typeAssertions.ts(31,12): error TS2737: Property 'p' is missing in type 'SomeOther' but present in type 'SomeBase'.

```

Comment:
Unclear if this is a regression. Are we simply overriding the head message?

Reply:
Yah. You mentioned wanting the top layer gone - that'd be simply overriding the head message.